src/metaswitch/clearwater/cluster_manager/plugin_utils.py

```diff
@@ -37,6 +38,31 @@ def write_cluster_settings(filename, cluster_view):
         f.write(new_file_contents)
 
 
+def write_chronos_cluster_settings(filename, cluster_view, current_server):
+    current_or_joining_servers = [constants.JOINING_ACKNOWLEDGED_CHANGE,
+                                  constants.JOINING_CONFIG_CHANGED,
+                                  constants.NORMAL_ACKNOWLEDGED_CHANGE,
+                                  constants.NORMAL_CONFIG_CHANGED,
+                                  constants.NORMAL]
+    leaving_servers = [constants.LEAVING_ACKNOWLEDGED_CHANGE,
+                       constants.LEAVING_CONFIG_CHANGED]
+
+    nodes = ([k for k, v in cluster_view.iteritems()
+              if v in current_or_joining_servers])
+    leaving = ([k for k, v in cluster_view.iteritems()
+               if v in leaving_servers])
+
+    with open(filename, 'w') as f:
+        f.write(dedent('''\
+        [cluster]
+        localhost = {current_server}
+        ''').format(**locals()))
```

Comment:
I'm not convinced that `**locals()` gains you much here  `"localhost = {}".format(current_server)` would work just as well and is a bit more obvious.